src/app/frontend/deploy/deployfromsettings.html

```diff
@@ -81,37 +86,38 @@
 
 <kd-help-section>
   <md-checkbox ng-model="ctrl.isExternal" class="md-primary"
-             ng-model-options="{ debounce: { 'default': 500, 'blur': 0 } }">
+               ng-model-options="{ debounce: { 'default': 500, 'blur': 0 } }">
     Expose service externally
   </md-checkbox>
 </kd-help-section>
 
+<!-- advanced options -->
 <div ng-show="ctrl.isMoreOptionsEnabled()">
   <kd-help-section>
-    <md-input-container>
+    <md-input-container class="md-block kd-deploy-block">
```

Comment:
IMHO, an input field is a global concept and should be styled globally. Of course, currently we only have one page. So, I agree `kd-deploy-block` is fine, but please refactor later...